cmake/Macros.cmake

```diff
@@ -257,44 +257,10 @@ macro(sfml_add_example target)
         target_link_libraries(${target} PRIVATE ${THIS_DEPENDS})
     endif()
 
-    set(target_install_dir ${SFML_MISC_INSTALL_PREFIX}/examples/${target})
-
-    if(BUILD_SHARED_LIBS AND (SFML_OS_LINUX OR SFML_OS_FREEBSD))
-        file(RELATIVE_PATH rel_lib_dir
-             ${CMAKE_INSTALL_PREFIX}/${target_install_dir}
-             ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
-
-        set_target_properties(${target} PROPERTIES
-                              INSTALL_RPATH "$ORIGIN/${rel_lib_dir}")
-    endif()
-
     if (SFML_OS_IOS)
         sfml_set_common_ios_properties(${target})
     endif()
 
-    # add the install rule
-    install(TARGETS ${target}
```

Comment:
Why are the examples not installed anymore?

Reply:
Most people don't want them to be installed, and they commonly aren't

Reply:
Not sure if this be done in one go, though... @eXpl0it3r, thoughts?

Reply:
I suggested to remove it, otherwise we now have to have a discussion where the examples should be installed to with the new GNU dirs, which then eventually will lead to the discussion about whether we should install them at all and then we can go round and round from there (see #189).

I'd much rather go the other way of removing the installation and let those, who feel that it's really needed, come up with a good directory suggestion. That way we won't be blocking this PR.

Reply:
Okay, fair enough.

Reply:
The install target is supposed to generate a SFML package similar to the official releases. I can live with the exception of examples, but we (@eXpl0it3r) now have to remember to do the extra step of copying examples when preparing SFML releases.